grandcypher/test_queries.py

```diff
@@ -1661,3 +1661,94 @@ def test_path(self, graph_type):
 
         res = GrandCypher(host).run(qry)
         assert len(res["P"][0]) == 5
+
+
+class TestMatchWithOrOperatorInRelationships:
+    @pytest.mark.parametrize("graph_type", ACCEPTED_GRAPH_TYPES)
+    def test_match_with_single_or_operator(self, graph_type):
+        host = graph_type()
+        host.add_node("a", name="Alice")
+        host.add_node("b", name="Bob")
+        host.add_node("c", name="Carol")
+        host.add_edge("a", "b", __labels__={"LOVES"})
+        host.add_edge("b", "c", __labels__={"WORKS_WITH"})
+
+        qry = """
+        MATCH (n1)-[r:LOVES|WORKS_WITH]->(n2)
```

Comment:
Alice and Bob should disclose this to HR

Reply:
😂